client/task-list/index.js

```diff
@@ -203,19 +203,11 @@ class TaskDashboard extends Component {
 		} );
 	}
 
-	keepTaskCard() {
```

Comment:
Was this method no longer being used?

Reply:
Ya, it seems like it was mistakenly left in there from an older iteration of onboarding.  It's not currently called anywhere.